src/transforms/addEslintDisableComment.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Type definitions for VSCode autocompletion!
+ *
+ * @typedef {Object} TransformParams
+ * @property {*} ast - The resulting AST as parsed by babel
+ * @property {import("ast-types/gen/builders").builders} builder - Recast builder for transforming the AST
+ * @property {*} options - Options passed into the transform from the CLI (if any)
+ */
+
+/**
+ * @param {TransformParams} param0
+ * @returns {import("ast-types").Visitor}
+ */
+
+const reactHookPrefix = new RegExp("use\\w+");
+const transform = ({ builder }) => {
+  return {
+    visitExportNamedDeclaration(path) {
+      const includesHook = path.node.specifiers.some(specifier => {
+        if (reactHookPrefix.test(specifier.local.name)) return true;
+      });
+
+      if (includesHook) {
+        // create exact copy of the export declaration
+        const newDeclaration = builder.exportNamedDeclaration(path.node.declaration, [
+          ...path.node.specifiers,
+        ]);
+        const newDeclarations = path.node.specifiers.map((_, index) => {
+          if (index === 0) {
+            newDeclaration.comments = [
+              builder.commentBlock("eslint-disable @fullscript/gql-no-manual-hook-declaration "),
```

Comment:
Something that I noticed when running this locally. ESlint seems to complain whenever there's no initial space in the comment. 

![image](https://github.com/Fullscript/js-transforms/assets/5891054/7fab74a9-ae8c-476b-b884-c84e7fede772)

```suggestion
              builder.commentBlock(" eslint-disable @fullscript/gql-no-manual-hook-declaration "),
```

Reply:
Nice catch!